src/frontend/src/lib/components/dapps/DappsCarousel.svelte

```diff
@@ -30,7 +38,7 @@
 	}: CustomEvent<CarouselSlideOisyDappDescription['id']>) => {
 		const idx = dappsCarouselSlides.findIndex(({ id }) => id === dappId);
 
-		hiddenDappsIds = [...hiddenDappsIds, dappId];
+		temporaryHiddenDappsIds = [...temporaryHiddenDappsIds, dappId];
 
 		dappsCarouselSlides = filterCarouselDapps({ dAppDescriptions, hiddenDappsIds });
```

Comment:
Do we still need to call it here? Because now it doesn't include the current `dappId`. I assume the `dappsCarouselSlides` value will be updated then because of line 32.

Reply:
We still do, unfort, to avoid that the reactive statement is not that quick, and the slide is removed and re-put...

And actually, now that you made me noticed it, I need to leave the update of `hiddenDappsIds` there...

Reply:
Ok, I see, thanks for the explanation!